tdrs-backend/tdpservice/data_files/views.py

```diff
@@ -132,11 +132,6 @@ def get_s3_versioning_id(self, file_name, prefix):
     def get_queryset(self):
         """Apply custom queryset filters."""
         queryset = super().get_queryset().order_by('-created_at')
-        if self.request.query_params.get('file_type') == 'ssp-moe':
```

Comment:
Not sure how this ever worked since we were never passing the `file_type` as a query param from the frontend. We filter our query set by Datafile.pk which is why we don't need the ssp filter.

Reply:
the same `get_queryset` is used by the auto-created `list` endpoint (part of the DRF `ModelViewSet` this class inherits from), and the `file_type` param is used on the frontend for that endpoint (`/data_files`). I think removing the filter entirely has potential side effects for users assigned to a location that can submit either `TANF` or `SSP-MOE` files. I'd suggest wrapping this block in `if self.action != 'list':` similar to how `filter_queryset()` works below.

Reply:
Very nice catch sir! I didn't see the incorrect behavior as I was switching between program types because I was also changing the year or quarter. Changing all three or two of the three always showed the correct results. However, when only changing the program type the error reared its head!

Reply:
@jtimpe tagging for re-review